static/js/stream_create.js

```diff
@@ -403,4 +403,12 @@ export function set_up_handlers() {
             e.preventDefault();
         }
     });
+
+    $(window).on("resize", () => {
+        if ($(window).width() <= 768) {
+            $(".stream-creation-body").css("height", "calc(95vh - 115px)");
+        } else {
+            $(".stream-creation-body").css("height", "calc(95vh - 155px)");
+        }
+    });
```

Comment:
This doesn't work if the web app is loaded at a width less than 768, instead of being resized. I guess CSS media queries would be helpful here.

Reply:
i changed this so now it uses css media queries